### PR TITLE
Fix and update WebLLM example

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Todo

- [x] Update LLama model. Try the update here https://py.cafe/panel-org/webllm.
- [x] Phillip: Fix wrong webllm link (#7350)

I don't know how to fix the broken link

### CODE REVIEW

**Comment by philippjfr** [comment]
Just lower case it.

File: examples/gallery/webllm.ipynb [hunk 1]
```diff
@@ -34,9 +34,9 @@
     "MODELS = {\n",
     "    'SmolLM (130MB)': 'SmolLM-135M-Instruct-q4f16_1-MLC',\n",
     "    'TinyLlama-1.1B-Chat (675 MB)': 'TinyLlama-1.1B-Chat-v1.0-q4f16_1-MLC-1k',\n",
-    "    'Gemma-2b (1895 MB)': 'gemma-2-2b-it-q4f16_1-MLC',\n",
-    "    'Mistral-7b-Instruct (4570 MB)': 'Mistral-7B-Instruct-v0.3-q4f16_1-MLC',\n",
-    "    'Llama-3.1-8b-Instruct (4598 MB)': 'Llama-3-8B-Instruct-q4f16_1-MLC-1k',\n",
+    "    'Gemma-2b (2GB)': 'gemma-2-2b-it-q4f16_1-MLC',\n",
+    "    'Mistral-7b-Instruct (5GB)': 'Mistral-7B-Instruct-v0.3-q4f16_1-MLC',\n",
+    "    'Llama-3.2-1B-Instruct (5GB)': 'Llama-3.2-1B-Instruct-q4f16_1-MLC',\n",
```
Comment (ahuang11) [inline examples/gallery/webllm.ipynb:39]: I don't think the 3.2 billion model is 5 GB.

**Comment by MarcSkovMadsen** [comment]
> Just lower case it.

But where? I could not figure out where.

**Comment by philippjfr** [comment]
doc/gallery/index.md I think

File: examples/gallery/webllm.ipynb (hunk 1 above)
Comment (philippjfr) [reply examples/gallery/webllm.ipynb:39]: Oh yeah, this isn't right at all.

**Pull request merged**